src/base_station/log_view.cpp

```diff
@@ -11,17 +11,20 @@ const int FONT_SIZE = 15;
 
 unsigned int chars_per_line = 32;
 unsigned int num_lines = 15;
+unsigned int total_lines = 1000;
+unsigned int view_index = 0;
```

Comment:
Might want to add a comment that says what this is, since its not totally clear from the variable name. The others here are self-explanatory, but `view_index` is not.